workflow/scripts/tmp_deeptools_plotfingerprint.py

```diff
@@ -0,0 +1,18 @@
+from snakemake.shell import shell
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+out_counts = snakemake.output.get("counts")
+
+optional_output = ""
+
+if out_counts:
+    optional_output += " --outRawCounts {out_counts} ".format(out_counts=out_counts)
+
+shell(
+    "(plotFingerprint "
+    "-b {snakemake.input.bam_files} "
+    "-o {snakemake.output.fingerprint} "
+    "{optional_output} "
+    "{snakemake.params}) {log}"
+)
```

Comment:
This file is obsolete with the new wrappers, right? Although I don't see it among the rules above, so maybe the wrapper to replace this file still needs to be added to this PR?